library/src/main/res/values/colors.xml

```diff
@@ -2,15 +2,10 @@
 
 <resources>
 	<color name="background_tab_pressed">#6633B5E5</color>
-	<color name="activate_text_color">#FF666666</color>
-	<color name="deactivate_text_color">#FFCCCCCC</color>
 	<color name="transparent">#00000000</color>
 	<color name="borderGrey">#c2c2c2</color>
 	<color name="spotlightIndicatorLightGrey">#767676</color>
-	<color name="headingText">#6d6d6d</color>
-	<color name="collectionItemLabelBackground">#E71324</color>
-	<color name="collectionItemImageBorder">#0a000000</color>
-	<color name="collectionItemActive">#FFFFFF</color>
+	<color name="arc_red">#E71324</color>
```

Comment:
Please use generic naming in this library. arc = specific client.

Reply:
Yes, I will 😃 Thanks Tim!